src/messages/manifestjson.js

```diff
@@ -298,12 +298,10 @@ export function corruptIconFile({ path }) {
   };
 }
 
-export const MANIFEST_THEME_LWT_ALIAS = {
-  code: 'MANIFEST_THEME_LWT_ALIAS',
-  message: i18n._('This theme LWT alias has been removed in Firefox 70.'),
-  description: i18n._(
-    'See https://mzl.la/2T11Lkc (MDN Docs) for more information.'
```

Comment:
So, yeah, a map is what I'm after right now.

Ideally, we'll be able to overwrite both, `message` and `description` too (`description` is being taken from the schema by default) so that we can provide a proper user experience.